frontend/src/Component/ApplicationGallary.tsx

```diff
@@ -2,20 +2,20 @@ import {Application} from "../types/Application.ts";
 import ApplicationCard from "./ApplicationCard.tsx";
 
 
-type ApplicationGalleryProps={
-    applications:Application[]
+type ApplicationGalleryProps = {
+    applications: Application[]
 
 
 }
-export default function ApplicationGallery({applications}: Readonly<ApplicationGalleryProps>){
-    const applicationCards=applications.map((application:Application)=>(<div  key={application.id}>
+export default function ApplicationGallery({applications}: Readonly<ApplicationGalleryProps>) {
+    const applicationCards = applications.map((application: Application) => (<div key={application.id}>
```

Comment:
This looks very much like a list. Highly recommend returning `li` elements instead of a `div` and wrapping it in a `ul` tag, when rendering later.